lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,29 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        if not qs:
```

Comment:
Have you considered to use decorator for this?

Reply:
@cmltaWt0 I quit can't see why and where a decorator is applicable here. Could you provide more details?

Reply:
Decorator to create `default_onboarding` in a different way.
Something like:
```python
@ensure_default_exists
def def get_queryset(self, request):
    return super(OnboardingAdmin, self).get_queryset(request)
```


It's just a thoughts :) Don't worry.

Reply:
ok, got it.